oojs.js

```diff
@@ -10,3 +10,23 @@
   // Have an artist property that is determined by some input.
 
 // Create an instance of the Album class.
+
+class Playlist{
+  constructor(title, song){
+    this.title = title;
+    this.song = song;
+    }
+  addSong(){
```

Comment:
`addSong` needs to take a parameter for the song you want to add, and there must be a push to the songs array:
```js

    addSong(song){
        this.songs.push(song)
    }
```